src/config.rs

```diff
@@ -210,6 +211,7 @@ impl Default for CosmosSection {
             key_derivation_path: "m/44'/118'/0'/0/0".to_owned(),
             prefix: "somm".to_owned(),
             msg_batch_size: 5,
+            gas_limit_per_msg: 20_000_000u64,
```

Comment:
The chain has since executed a params proposal that raised this limit to 60M I believe.